clang/runtime/dpct-rt/include/device.hpp

```diff
@@ -64,22 +64,22 @@ class device_info {
   // get interface
   const char *get_name() const { return _name; }
   char *get_name() { return _name; }
-  template <typename WorkItemSizesTy = sycl::id<3>,
-            std::enable_if_t<std::is_same_v<WorkItemSizesTy, sycl::id<3>> ||
+  template <typename WorkItemSizesTy = sycl::range<3>,
```

Comment:
Please add back old interface for sycl::id,  at same time add "deprecated"  keywords.

Reply:
not needed, sycl::id can be implicitly converted by sycl::range